tasks/system/forwarding.yml

```diff
@@ -8,3 +8,12 @@
     state: present
     reload: true
   when: not lookup('env', 'IN_MOLECULE') | d(true, true) | bool
+
+- name: Set IPv6 forwarding in the sysctl file and reload if necessary
```

Comment:
Could you also rename the previous task to say `IPv4`?

Reply:
Done in c1cbc8c